packages/@expo/cli/src/start/server/metro/__tests__/createExpoMetroResolver.test.ts

```diff
@@ -22,7 +22,7 @@ const createContext = ({
   nodeModulesPaths?: string[];
   packageExports?: boolean;
   override?: Partial<SupportedContext>;
-}): SupportedContext => {
+}): SupportedContext & { unstable_fileSystemLookup?: (filepath: string) => any } => {
```

Comment:
We can make this typed better through the new Metro types 
<img width="703" alt="image" src="https://github.com/user-attachments/assets/91ef452b-759f-4a5d-a4e1-57811e31ae7c">
<img width="817" alt="image" src="https://github.com/user-attachments/assets/f1fb5a6b-400b-43e7-ab99-f0b21869cba3">